### PR TITLE
Saved both raw and processed textract output

### PR DESCRIPTION
**Pull request by antidipyramid**
## Overview

Textract's processed output seems to be missing some form key-value data from the PDF. This PR saves both the processed and raw response from Textract to help debug the issue.

`scripts/process_pdfs.py` generates two JSON files for each PDF: the raw response from Amazon Textract and the processed key-value form data. Respectively, they are stored in `textract_responses/` and `final/`.

## Testing Instructions

- Navigate to `final/` and examine the script's output

Handles #10

### CODE REVIEW

File: scripts/process_pdfs.py [hunk 1]
```diff
@@ -40,7 +40,10 @@ def main():
         form_key_values = textract.get_kv_relationship(key_map, value_map, block_map)
 
         print('Writing output...')
-        with open(f'final/{pdf_name}.json', 'w') as output:
+        with open(f'final/{pdf_name}_raw.json', 'w') as output:
+            output.write(json.dumps(all_blocks))
+
+        with open(f'final/{pdf_name}_processed.json', 'w') as output:
```
Comment (fgregg) [inline scripts/process_pdfs.py:46]: can we write these to an intermediate directory, and then we can keep the sames without the _raw or _processed suffixes.

**Review by fgregg — CHANGES_REQUESTED** [review]

File: scripts/process_pdfs.py (hunk 1 above)
Comment (antidipyramid) [reply scripts/process_pdfs.py:46]: Do we want a these files inside of `final/` (i.e. `final/raw/` and `final/processed/`)?

Or we could put a directory at the root similar to `images/` and store the raw files there?

Comment (fgregg) [reply scripts/process_pdfs.py:46]: > Or we could put a directory at the root similar to images/ and store the raw files there?

yup!

File: scripts/process_pdfs.py [hunk 2]
```diff
@@ -40,6 +40,9 @@ def main():
         form_key_values = textract.get_kv_relationship(key_map, value_map, block_map)
 
         print('Writing output...')
+        with open(f'textract_responses/{pdf_name}.json', 'w') as output:
+            output.write(json.dumps(all_blocks))
```
Comment (fgregg) [inline scripts/process_pdfs.py:44]:
```suggestion
            json.dump(all_blocks, output, indent=4)
```

File: scripts/process_pdfs.py [hunk 3]
```diff
@@ -40,6 +40,9 @@ def main():
...
+
         with open(f'final/{pdf_name}.json', 'w') as output:
             output.write(json.dumps(form_key_values))
```
Comment (fgregg) [inline scripts/process_pdfs.py:47]:
```suggestion
            json.dump(form_key_values, output, indent=4)
```

**Review by fgregg — CHANGES_REQUESTED** [review]
just a small changes to write more directly, and do pretty printing

**Review by fgregg — CHANGES_REQUESTED** [review]
a more direct way to write output and pretty printing

**Review by fgregg — APPROVED** [review]

**Comment by fgregg** [comment]
thanks @antidipyramid! I'll handle the merge conflict!

**Pull request merged**